### PR TITLE
Configfril

### PR DESCRIPTION
**Pull request by mlos100**
Add ability to create a FRIL matching rule configuration file and direct FRIL to use that configuration file and perform a deduplication.

### CODE REVIEW

File: examples/ptmatchadapter-fril/src/main/java/org/mitre/ptmatchadapter/RecordMatchRequestProcessor.java [hunk 1]
```diff
@@ -431,6 +597,45 @@ private void writeData(File f, List<Resource> resources, String serverBase)
...
+        .getResourceAsStream(name);
+
+    if (instream == null) {
+      instream = System.class.getResourceAsStream(name);
+    }
+    if (instream == null) {
+      try {
+        instream = new FileInputStream(name);
+      } catch (FileNotFoundException e) {
+        // Ignore
```
Comment (eedrummer) [inline examples/ptmatchadapter-fril/src/main/java/org/mitre/ptmatchadapter/RecordMatchRequestProcessor.java:617]: Eek. At least log or rethrow as a RuntimeException

File: examples/ptmatchadapter-fril/src/main/java/org/mitre/ptmatchadapter/RecordMatchRequestProcessor.java [hunk 2]
```diff
@@ -130,57 +159,64 @@ public void process(Bundle bundle) {
...
-          // retrieve other pages of search results
-          Bundle nextResults = masterSetResults;
-          while (nextResults.getLink(Bundle.LINK_NEXT) != null) {
-            nextResults = fhirRestClient.loadPage().next(nextResults).execute();
-            List<Resource> nextResources = resultSplitter.splitBundle(nextResults);
-            writeData(masterFile, nextResources, masterServerBase);
-          }
+          retrieveAndStoreData(masterSearchUrl, masterServerBase, jobDir, "master");
+
+          // IQuery<Bundle> query =
```
Comment (eedrummer) [inline examples/ptmatchadapter-fril/src/main/java/org/mitre/ptmatchadapter/RecordMatchRequestProcessor.java:167]: Should be removed in a future commit

**Comment by eedrummer** [comment]
Looks reasonable. A few comments left on the code for future consideration.

**Comment by mlos100** [comment]
Both comments addressed in forthcoming buildresults branch

Mike

**Pull request merged**